nprime.js

```diff
@@ -0,0 +1,30 @@
+// program to print first n prime numbers
+
+// take input from the user
+const num = parseInt(prompt('Enter the number of primes required: '));
+
+
+//define prime
+const isPrime = (n) => {
+    for(let i = 2; i <= n/2; i++){
```

Comment:
2 spaces instead of 4

Reply:
kindly mention the line, I am bit confused about which spaces are u talking

Reply:
I am talking about indentation. 
u can use [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)